src/script/connection/ConnectionRepository.ts

```diff
@@ -311,6 +318,39 @@ export class ConnectionRepository {
     } catch (error) {
       const logMessage = `Connection change from '${currentStatus}' to '${newStatus}' failed`;
       this.logger.error(`${logMessage} for '${userEntity.id}' failed: ${error.message}`, error);
+      switch (newStatus) {
+        case ConnectionStatus.ACCEPTED: {
+          PrimaryModal.show(PrimaryModal.type.ACKNOWLEDGE, {
+            text: {
+              htmlMessage: t('modalUserCannotAcceptConnectionMessage'),
+              title: t('modalUserCannotConnectHeadline'),
+            },
+          });
+        }
+        case ConnectionStatus.CANCELLED: {
+          PrimaryModal.show(PrimaryModal.type.ACKNOWLEDGE, {
+            text: {
+              htmlMessage: t('modalUserCannotCancelConnectionMessage'),
+              title: t('modalUserCannotConnectHeadline'),
+            },
+          });
+        }
+        case ConnectionStatus.IGNORED: {
+          PrimaryModal.show(PrimaryModal.type.ACKNOWLEDGE, {
+            text: {
+              htmlMessage: t('modalUserCannotIgnoreConnectionMessage'),
+              title: t('modalUserCannotConnectHeadline'),
+            },
+          });
+        }
+        default: {
+          PrimaryModal.show(PrimaryModal.type.ACKNOWLEDGE, {
+            text: {
+              title: t('modalUserCannotConnectHeadline'),
+            },
```

Comment:
this shows a modal with no message, only the title "Connecting not possible" as a fallback for unhandled cases

Reply:
Doesn't it Look weird if the modal doesn't have a message?

Reply:
looks fine to me, wdyt?
![image](https://github.com/wireapp/wire-webapp/assets/78490891/5bbf3110-c3bc-4d26-b9d6-c4b178fedafe)